FirebaseAuthUI/README.md

```diff
@@ -69,8 +70,9 @@ authUI?.delegate = self
 @import FirebaseAuthUI
 ...
 [FIRApp configure];
-FIRAuthUI *authUI = [FIRAuthUI defaultAuthUI];
-authUI.delegate = self; // Set the delegate to receive callback.
+FIRAuthUI *authUI = [FIRAuthUI default];
```

Comment:
Thanks you @morganchen12. Where can I get a clue about objc import rule? I can not find out NS_SWIFT_NAME in FIRAuthUI.h source code(0.6.1 release source files).